### ISSUE TITLE
"Rebrand" the extension to work with the Validator as well

### ISSUE THREAD

**Issue by marko-bekhta** [hibernate/hibernate-develocity-maven-extension#28]
I had a quick check on integrating Develocity with Validator. From what it seems, most of the things would work as is from the extension perspective, except  maybe adjusting the tag here:

https://github.com/hibernate/hibernate-search-develocity-extension/blob/6c369171caecdcd7a3539d1edc92df10d3234299/src/main/java/org/hibernate/search/develocity/scan/BuildScanMetadata.java#L38

maybe to something like 
```java
// e.g. "org.hibernate.search" or "org.hibernate.validator"
buildScanApi.tag( currentProject.getModel().getGroupId() );
```

and then doing some renaming... maybe: 
```xml
    <groupId>org.hibernate.infra.develocity</groupId>
    <artifactId>hibernate-develocity-maven-extension</artifactId>
```

and here's a link to one of the local builds https://ge.hibernate.org/s/n7r6spbey4wrc (when most of the stuff got cached)

**Comment by marko-bekhta** [comment]
@yrodiere let me know if this makes sense to you 🙈

**Comment by yrodiere** [comment]
I missed this notification, sorry... everything you said LGTM, please go ahead :+1:

**Comment by yrodiere** [comment]
@marko-bekhta did you look through the code though? I see a few things that look very specific to Search, e.g. this:

https://github.com/hibernate/hibernate-search-develocity-extension/blob/6c369171caecdcd7a3539d1edc92df10d3234299/src/main/java/org/hibernate/search/develocity/plugins/ForbiddenApisConfiguredPlugin.java#L42-L46

**Comment by marko-bekhta** [comment]
yeah ... I looked at the other plugins and was thinking: ooook validator does not use these properties so it won't get into these conditions when running there ... but I missed this one 🙈 😕 thanks for pointing to it. Coincidentally (or is it 😏), Validator also uses the build config artefact to share additional forbiddenapis files, so it turned out to be a matter of pointing to the correct artifact depending on what we are building.
So I went with checking the group ids and then doing the additional checks per "project": https://github.com/hibernate/hibernate-search-develocity-extension/pull/29

**Comment by yrodiere** [comment]
Fixed by #29 

Follow-up work:

> * do a 3.0.0 release. 
> * update Search/Validator to use the new extension version
